ADOPTERS.md

```diff
@@ -60,6 +60,7 @@ to share the details of their usage publicly at this time.
 * [FHE3 GmbH](https://www.fhe3.com/)
 * [infraBuilder](https://infrabuilder.com/)
 * [GreenCom Networks](http://www.greencom-networks.com/en)
+* [PITS Global Data Recovery Services](https://www.pitsdatarecovery.net/)
```

Comment:
There are two sections of this doc, first with a description and then without. If you would like to add a brief description of your usage, it would be great to have more details and move you up to that section.